docs/ldml/tr35-keyboards.md

```diff
@@ -1163,6 +1167,119 @@ This attribute may be escaped with `\u` notation, see [Escaping](#Escaping).
 
 * * *
 
+### <a name="Element_forms" href="#Element_forms">Element: forms</a>
+
+This element represents a set of `form` elements which define the layout of a particular hardware form.
+
+
+> <small>
+>
+> Parents: [keyboard](#Element_keyboard)
+>
+> Children: [import](#Element_import), [form](#Element_form), [_special_](tr35.md#special)
+>
+> Occurrence: optional, single
+>
+> </small>
+
+***Syntax***
+
+```xml
+<forms>
+    <form id="iso">
+        <!-- ... -->
+    </form>
+    <form id="us">
+        <!-- ... -->
+    </form>
+</forms>
+```
+
+* * *
+
+### <a name="Element_form" href="#Element_form">Element: form</a>
+
+This element represents a specific `form` element which defines the layout of a particular hardware form.
```

Comment:
The word "layout" is a bit loaded, maybe we could use "arrangment" for the physical layout of keys?